scripts/protocgen.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -eo pipefail
+
+echo "Generating gogo proto code"
```

Comment:
are we using buf to generate the proto files now?
is this the same as ignite?
if not why is this better?